### ISSUE TITLE
How to build and run?

### ISSUE THREAD

**Issue by kgn** [BohemianCoding/SketchAPI#13]
Hey Guys, wondering about documentation about how to build and run this so that when we work on changes and want to submit pull requests we can test the changes in the Sketch app?

**Comment by kgn** [comment]
Is it safe to replace js scripts in `/Applications/Sketch.app/Contents/Resources/`?

**Comment by kgn** [comment]
Looks like it's not, if I make any changes to this file Sketch fails to launch. Guessing it fails some checksum of assets?

**Comment by bomberstudios** [comment]
You can run this in Terminal:

```
defaults write com.bohemiancoding.sketch3 SketchAPILocation "/path/to/your/SketchAPI.js"
```

and it _should_ read the .js file from the path you set.

**Comment by bomberstudios** [comment]
As for the build process itself, I guess we need to add some docs and update `gulpfile.js` to support standalone builds (at the moment it assumes it's built inside the Sketch repo, which won't be the case for 99.99% of users : )

**Comment by bomberstudios** [comment]
Added some notes about the build process in README.md, so closing : )

**Comment by mossyblog** [comment]
the default write doesn't work. I've checked the pathing etc and nothing

**Comment by WCByrne** [comment]
I seem to be missing something here. Built the API, set `path/to/SketchAPI.js` in defaults, but the modified API doesn't seem to be used. Is there a way to validate the alternate API path is being used. As a quick attempt I threw a `throw Error()` in a function (which I can see in the built SketchAPI.js) and tried to call it but it's not throwing.

**Comment by WCByrne** [comment]
@mathieudutour, sorry to ping you but it look like this was before your time. 

Is the above (and the docs) still accurate? I built the API, set `path/to/SketchAPI.js` in defaults, but the modified API doesn't seem to be used. Is there a way to validate the alternate API path is being used. As a quick attempt I threw a `throw Error()` in a function (which I can see in the built SketchAPI.js) and tried to call it but it's not throwing.

I've also confirmed that the default is set.
